R/avg.R

```diff
@@ -55,6 +55,10 @@ avg_event <- function(id, group, par = NULL) {
       function(x) sd(x$Freq, na.rm = TRUE) / sqrt(nrow(x)),
       FUN.VALUE = numeric(1)
     )
+    count <- vapply(split(res, res$Var2),
+      nrow,
```

Comment:
This should be the of event counts per treatment group, right? If so, the following is suggested:
```suggestion
      function(x) sum(x$Freq, na.rm = TRUE), 
```
Currently, this seems to be total subject counts.